transport/server.cc

```diff
@@ -12,7 +12,9 @@
 #include "cql3/statements/modification_statement.hh"
 #include <seastar/core/scheduling.hh>
 #include <seastar/core/semaphore.hh>
+#include "seastar/core/sstring.hh"
 #include "seastar/coroutine/switch_to.hh"
+#include "transport/event.hh"
```

Comment:
Nit: not sure if adding these hearders is strictly necessary. Moreover, we include seastar headers via angle brackets.